config/resources/publicatie-domain.lisp

```diff
@@ -80,4 +80,18 @@
   :features '(include-uri)
   :on-path "publication-types")
 
+
+(define-resource regulation-type () ;; maybe legislation ? is this the correct name for this?
+  :class (s-prefix "ext:RegelgevingType") ;; used ext instead of pub because this could be used on cases
+  :properties `((:label             :string ,(s-prefix "skos:prefLabel"))
+                (:scope-note        :string ,(s-prefix "skos:scopeNote"))
+                (:priority          :number ,(s-prefix "ext:priority"))
```

Comment:
Kunnen we voor de prioriteit property `https://schema.org/position` gebruiken?

Reply:
We gebruiken nu al ext:priority en ext:prioriteit (zouden we ook eens gelijk moeten trekken)
Willen we daar nu een 3de type introduceren voor hetzelfde?
De omschrijving van deze position predicate is ook eerder in functie van sequentiële items.
Zoals in "Doe eerst dit, dan dit, dan dat".
Is dat hetzelfde als prioriteit in deze context, wat we eigenlijk gebruiken om de data te kunnen sorteren?

Echt lijsten hebben met volgorde kan in rdf, maar zit wat complexer in elkaar en dit willen we waarschijnlijk vermijden door een getal op te slagen in ons datamodel.
https://ontola.io/blog/ordered-data-in-rdf/

Reply:
'Echte' lijsten in RDF zijn gemodelleerd als linked list en dat willen we inderdaad vermijden, want niet praktisch bruikbaar om weer te geven in een GUI.

Met `schema:position` druk je de positie van een item in een lijst uit. En dat lijkt mij exact wat we hier willen zeggen. De regelgevingstypes moeten in een bepaalde vooraf vastgelegde volgorde weergegeven worden. Ik vermoed voornamelijk voor gebruiksgemak zodat dat de meeste gebruikte eerst staan. Met `ext:priority` wordt er geimpliceerd dat het ene regelgevingstype belangrijker is dan een ander type. Dat lijkt me hier niet het geval.

Het gebruik van `ext:priority`  vs `ext:prioriteit` moeten we ook eens rechttrekken - niet in de scope van deze PR - maar ik denk dat er daar ook heel wat gaan kunnen vervangen worden door `schema:position` omdat ze eigenlijk enkel een volgorde in een lijst uitdrukken en geen priorititeit (x is belangrijker dan y).